bundles/org.openhab.binding.knx/src/test/java/org/openhab/binding/knx/internal/dpt/DPTTest.java

```diff
@@ -123,6 +123,10 @@ void testToDPT7ValueFromQuantityType() {
         assertEquals("1000", ValueEncoder.encode(new QuantityType<>("1000 lx"), "7.013"));
 
         assertEquals("3000", ValueEncoder.encode(new QuantityType<>("3000 K"), "7.600"));
+        // unit for 7.600 is K; special handling for color temperature: make sure °C and mired work as well
+        assertEquals("3273.15", ValueEncoder.encode(new QuantityType<>("3000 °C"), "7.600"));
+        assertEquals("4000", ValueEncoder.encode(new QuantityType<>("250 mired"), "7.600"));
+        assertEquals("4000", ValueEncoder.encode(new QuantityType<>("250 mirek"), "7.600"));
```

Comment:
Not strictly necessary to test it (the aliases are anyway tested in core) .. but for completeness MK⁻¹ is another possible alias.